src/pages/ProfileScreen.js

```diff
@@ -1,52 +1,70 @@
 import React from 'react';
 import { StyleSheet, Text, View, TextInput, Image, Button } from 'react-native';
-import { Header } from 'react-native-elements'
+import Icon from 'react-native-vector-icons/FontAwesome';
+import Prompt from 'react-native-prompt-crossplatform';
```

Comment:
Do you really need to use it? Think carefully before trying to add some new library. Otherwise it'll make the app size bigger for no reason.